src/super_gradients/scripts/merge_recipe.py

```diff
@@ -0,0 +1,47 @@
+"""
+Script that saves a merged (i.e no inheritance from other yaml configuration files),
+ .yaml file that can be ran on its own without the need to keep other configurations which the original
+  file inherits from.
+
+Usage:
+    python merge_recipe --config_name=cifar10_resnet -> saves cifar10_resnet_complete.yaml in path/to/super_gradients/recipes
```

Comment:
Just a 5 cents regarding naming.

I suggest using `export_recipe` name instead of `merge_recipe`. From my perspective `merge` verb represents the internal implementation details of **what** happens under the hood and not the **user** intention. 

Also, `merge` usually means you have two or more items of the same "quality" to merge (E.g two top-level configs), which is not the case here. What happens is config composition and not merge, strictly speaking.

Reply:
Was feeling unease with merge TBH.
Changed in d08558ed5b14b7c26502fcb0ffcbdda6afb48ead